tests/test_api.py

```diff
@@ -102,6 +102,18 @@ async def test_api_get_resource(db, client, insert_fake_resource):
     data: dict = await resp.json()
     assert data["dataset_id"] == DATASET_ID
     assert data["resource_id"] == RESOURCE_ID
+    assert data["status"] == "TO_CHECK"
+
+
+@pytest.mark.parametrize("resource_status", list(Resource.STATUSES.items()))
+async def test_api_get_resource_status(db, client, insert_fake_resource, resource_status):
```

Comment:
If I understand [pytest parametrization correctly](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions), you could even:

```suggestion
@pytest.mark.parametrize("resource_status,resource_verbose_status", list(Resource.STATUSES.items()))
async def test_api_get_resource_status(db, client, insert_fake_resource, resource_status, resource_verbose_status):
```
and then use `resource_status` instead of `resource_status[0]`, and `resource_verbose_status` instead of `resource_status[1]`, i think?

Reply:
Good idea, it's easier to read! Thanks! Done